docs/reference/tab-widgets/troubleshooting/data/increase-cluster-shard-limit.asciidoc

```diff
@@ -0,0 +1,210 @@
+//////////////////////////
+
+[source,console]
+--------------------------------------------------
+PUT my-index-000001
+
+--------------------------------------------------
+// TESTSETUP
+
+[source,console]
+--------------------------------------------------
+PUT _cluster/settings
+{
+  "persistent" : {
+    "cluster.routing.allocation.total_shards_per_node" : null
+  }
+}
+
+DELETE my-index-000001
+--------------------------------------------------
+// TEARDOWN
+
+//////////////////////////
+
+// tag::cloud[]
+In order to get the shards assigned we'll need to increase the number of shards 
+that can be collocated on a node in the cluster in order to get all shards assigned. 
```

Comment:
This begins and ends with "in order to get all/the shards assigned"; it should go either at the beginning or the end, but not both (I think only the beginning).

```suggestion
In order to get the shards assigned we'll need to increase the number of shards 
that can be collocated on a node in the cluster.
```